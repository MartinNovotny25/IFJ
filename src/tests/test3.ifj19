def foo():
    x = 1
    return x
def main():
    # assigment
    x, y, z = 'foo', 14, 21.1
    print(x, y, z)

    # for i in range(start, stop, step)
    while True:
        print('1')

    #

    # function call
    y = foo()

if __name__ == '__main__':
    # main
    main()
