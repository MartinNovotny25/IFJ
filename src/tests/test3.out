TOKEN(KLICOVE_SLOVO)
TOKEN(IDENTIFIKATOR)
TOKEN(()
TOKEN())
TOKEN(:)
TOKEN(EOL)
TOKEN(INDENT)
TOKEN(IDENTIFIKATOR)
TOKEN(=)
TOKEN(INTEGER, 1)
TOKEN(EOL)
TOKEN(KLICOVE_SLOVO)
TOKEN(IDENTIFIKATOR)
TOKEN(EOL)
TOKEN(DEDENT)
TOKEN(KLICOVE_SLOVO)
TOKEN(IDENTIFIKATOR)
TOKEN(()
TOKEN())
TOKEN(:)
TOKEN(EOL)
TOKEN(EOL)
TOKEN(INDENT)
TOKEN(IDENTIFIKATOR)
TOKEN(,)
TOKEN(IDENTIFIKATOR)
TOKEN(,)
TOKEN(IDENTIFIKATOR)
TOKEN(=)
TOKEN(STRING, foo)
TOKEN(,)
TOKEN(INTEGER, 14)
TOKEN(,)
TOKEN(DOUBLE, 21.100000)
TOKEN(EOL)
TOKEN(IDENTIFIKATOR)
TOKEN(()
TOKEN(IDENTIFIKATOR)
TOKEN(,)
TOKEN(IDENTIFIKATOR)
TOKEN(,)
TOKEN(IDENTIFIKATOR)
TOKEN())
TOKEN(EOL)
TOKEN(EOL)
TOKEN(EOL)
TOKEN(KLICOVE_SLOVO)
TOKEN(IDENTIFIKATOR)
TOKEN(:)
TOKEN(EOL)
TOKEN(INDENT)
TOKEN(IDENTIFIKATOR)
TOKEN(()
TOKEN(STRING, 1)
TOKEN())
TOKEN(EOL)
TOKEN(EOL)
TOKEN(EOL)
TOKEN(EOL)
TOKEN(EOL)
TOKEN(DEDENT)
TOKEN(IDENTIFIKATOR)
TOKEN(=)
TOKEN(IDENTIFIKATOR)
TOKEN(()
TOKEN())
TOKEN(EOL)
TOKEN(EOL)
TOKEN(DEDENT)
TOKEN(KLICOVE_SLOVO)
TOKEN(IDENTIFIKATOR)
TOKEN(==)
TOKEN(STRING, __main__)
TOKEN(:)
TOKEN(EOL)
TOKEN(EOL)
TOKEN(INDENT)
TOKEN(IDENTIFIKATOR)
TOKEN(()
TOKEN())
TOKEN(EOL)
TOKEN(DEDENT)
TOKEN(EOF)
